e2e/suite_test.go

```diff
@@ -119,27 +119,27 @@ func runScenario(ctx context.Context, t *testing.T, r *mrand.Rand, opts *scenari
 	if err != nil {
 		vmssSucceeded = false
 		if !isVMExtensionProvisioningError(err) {
-			t.Fatalf("encountered an unknown error while creating VM: %v", err)
+			t.Fatalf("encountered an unknown error while creating VM %s: %v", vmssName, err)
```

Comment:
FatalF will perform the same as `panic` no ? should we focus at levering one or the other ?

Reply:
Changed.